app/models/concerns/benchmark_searching.rb

```diff
@@ -104,6 +104,17 @@ def supported_os_minor_search(_filter, _operator, value)
         parameter: parameters }
     end
 
+    # Returns all supported profiles for a given OS major version
+    # through a benchmark referencing this OS major
+    def supported_profiles
+      Profile.canonical.joins(:benchmark)
+             .where(benchmarks: { ref_id: bm.ref_id })
```

Comment:
Are all benchmarks we store supported? Or should it also have some kind of safeguard based on the versions in SupportedSsg for that given OS major.

Reply:
@vkrizan :point_up:

Reply:
The latter. Supported benchmarks are those listed in SupportedSsg.